packages/components/src/toolbar-button/README.md

```diff
@@ -0,0 +1,26 @@
+# ToolbarButton
+
+A ToolbarButton can be used to add actions to your block control, usually inside a ToolbarGroup. It has similar features to the [Button](https://developer.wordpress.org/block-editor/components/button/) component. Using `ToolbarButton` will ensure the correct styling for a button in a toolbar, and also that keyboard interactions in a toolbar are consistent with the [WAI ARIA toolbar pattern](https://www.w3.org/TR/wai-aria-practices/#toolbar).
+
+## Usage
+
+```jsx
+import { ToolbarButton } from "@wordpress/components";
+import { edit } from "@wordpress/icons";
+ 
+const MyToolbarButton = () => (
+    <MyToolbarButton
+        title="Edit"
+        icon={ edit }
+        onClick={ onEdit } />
+);
+```
+
+## Props
+
+This component accepts [the same API of the Button](https://developer.wordpress.org/block-editor/components/button/#props) component.
```

Comment:
For linking to components, you want to not include the full domain, since the docs are browsed in multiple contexts. See [the Using Links section](https://developer.wordpress.org/block-editor/contributors/document/#using-links) of the contributing docs.

```suggestion
This component accepts [the same API of the Button](/packages/components/src/button/README.md#props) component.
```